### DISCUSSION TITLE
Is it intentional that the last row of a j-column is also justified?

### DISCUSSION THREAD

**Discussion by dahosek** [lvjr/tabularray#195] [Q&A]
```
\begin{tblr}{jll}
  \SetColumn{font=\sffamily, wd=1in} 
  \SetRow{font=\bfseries, valign=f}
  Some text that's very long 
  and runs long enough to show the
  justification 
        &{Long\\Header} & More\\
  \hline
  One & Red            & Blue \\
  Two & Black          & White
\end{tblr}
```
output:

<img width="365" alt="Screen Shot 2022-02-07 at 13 24 46" src="https://user-images.githubusercontent.com/13854707/152857558-f0b61f48-e260-4d67-a495-5e38abf0a98e.png">

**Comment by lvjr** [comment]
Since you didn't provide a complete example, I don't know the reason. With version 2021Q and the following code I got correct result.
```latex
\documentclass{article}
\usepackage{tabularray}
\begin{document}
\begin{tblr}{jll}
  \SetColumn{font=\sffamily, wd=1in} 
  \SetRow{font=\bfseries, valign=f}
  Some text that's very long 
  and runs long enough to show the
  justification 
        &{Long\\Header} & More\\
  \hline
  One & Red            & Blue \\
  Two & Black          & White
\end{tblr}
\end{document}
```
![image](https://user-images.githubusercontent.com/4239949/152896865-ab6095e7-4687-448f-9f1a-bfa04ec357bb.png)

**Reply by dahosek** [reply]
Hmm, a bit more digging indicates that there's some conflict in my document class (custom class of my own) that's to blame. Now I've got to dig into this to see what's responsible for the problem.

**Reply by dahosek** [reply]
OK, this is really weird: I get the issue if I load `fontspec` and set the main font to `Ehrhardt MT Std`. So if I do:

```
\usepackage{fontspec}
\setmainfont{Ehrhardt MT Std}

```

I get the result shown in my example (even if the table is not actually set in Ehrhardt), but if I do

```
\usepackage{fontspec}
\setmainfont{Times}
```

it does give a correct last line. Strangely enough, changing the width to 1.5in also gives the correct result. At this point, I'll just have to mark it as a mystery of nature.

**Reply by lvjr** [reply]
I think it is not a issue of `tabularray`. You could test the problem with a `parbox` or `minipage`.

**Reply by dahosek** [reply]
I suspect it might actually be an issue with fontspec doing weird things to inter-word spacing when new fonts are loaded. It's something to investigate, but a low priority for me. I think my initial example created the illusion of parfillskip=0pt because after line breaking, the last line had to shrink the inter-word spacing to fit on the line. IIRC, word spacing should be set at the font level, but I'm realizing I don't know exactly what happens in a non-DEK TeX engine for this.
